stores/yaml/store.go

```diff
@@ -204,9 +208,9 @@ func (store *Store) appendSequence(in []interface{}, sequence *yaml.Node) {
 	}
 	if len(comments) > 0 {
 		if beginning {
-			comments = store.addCommentsHead(sequence, comments)
+			store.addCommentsHead(sequence, comments)
```

Comment:
The return value is always `nil`. It's mainly returned so you can write `comments = store.addCommentsHead(xx, comments)` and continue using `comments` later on. But we have no "later" here, so removing the assignment is best (and makes the linter happy).